src/main.rs

```diff
@@ -120,16 +120,26 @@ async fn execute() -> Result<std::process::ExitCode> {
 
                     return Err(Error::Unauthorized(message))?;
                 }
-                _ => {
+                StatusCode::BAD_REQUEST => {
                     let body = response.bytes().await?;
                     let message = serde_json::from_slice::<String>(&body)?;
```

Comment:
Instead of trying to deserialize it as JSON, we could just deserialize it as a real String using `response.text().await` instead of `response.bytes().await`